docs/src/rules/valid-typeof.md

```diff
@@ -25,7 +25,7 @@ Examples of **incorrect** code for this rule:
 typeof foo === "strnig"
 typeof foo == "undefimed"
 typeof bar != "nunber"
-typeof bar !== "fucntion"
+typeof bar !== "function"
```

Comment:
```suggestion
typeof bar !== "fucntion"
```
This typo is correct here as its an incorrect example to show that typeof comparison must include valid strings